src/PropertyAppTests/JsonReadWriteTests/JsonHandlerTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using PropertyApp.JsonReadWrite;
+using PropertyApp.Model;
+using System.Diagnostics.CodeAnalysis;
+
+namespace PropertyAppTests.JsonReadWriteTests;
+
+[ExcludeFromCodeCoverage]
+[TestFixture]
+public class JsonHandlerTests
+{
+    private JsonHandler<IssueModel> _jsonHandler;
+    private readonly string _folderPath = Path.Combine($"{AppDomain.CurrentDomain.BaseDirectory}", @"..\..\..\JsonReadWriteTests");
+
+    [Test]
+    public void GetByIdGetsTheDesiredModel()
+    {
+        // Arrange
+        _jsonHandler = new JsonHandler<IssueModel>(_folderPath);
```

Comment:
`_jsonHandler` is used in three tests and is given the same ctor argument each time so should be in `SetUp()` method